numbers-booleans-objects/accessing-data-in-js-objects/README.md

```diff
@@ -0,0 +1,47 @@
+# Text
+
+Every Javascript objects has properties associated with it. And there are two ways to access those properties: 1) using `dot notation` and 2) using `bracket notation`.
+
+#### Dot notation
+```js
+  objectName.propertyName
+```
+
+```js
+  // Let's define a object
+  const myCar = {
+    make: 'Ford',
+    model: 'Aspire',
+    year: 2020
+  };
+
+  myCar.model // Returns: Aspire
+  myCar.year // Returns: 2020
+```
+
+When we try to access a property on object which is not defined yet, it returns `undefined`. For example:
+
+```js
+myCar.color; // undefined
+```
+
+#### Bracket notation
+In Javascript, Objects are sometimes called associative arrays, since each property is associated with a string value that can be used to access it. So, for example, you could access the properties of the **myCar** object like:
```

Comment:
```suggestion
In JavaScript, objects are sometimes called associative arrays, since each property is associated with a string value that can be used to access it. So, for example, you could access the properties of the **myCar** object like:
```